src/Microsoft.TestPlatform.CommunicationUtilities/DataCollectionTestCaseEventHandler.cs

```diff
@@ -98,18 +95,12 @@ public void ProcessRequests()
 
                     _communicationManager.SendMessage(MessageType.DataCollectionTestStartAck);
 
-                    if (EqtTrace.IsInfoEnabled)
-                    {
-                        EqtTrace.Info("DataCollectionTestCaseEventHandler: Test case '{0} - {1}' started.", testCaseStartEventArgs.TestCaseName, testCaseStartEventArgs.TestCaseId);
-                    }
+                    EqtTrace.Info("DataCollectionTestCaseEventHandler: Test case '{0} - {1}' started.", testCaseStartEventArgs?.TestCaseName, testCaseStartEventArgs?.TestCaseId);
```

Comment:
Do we want to have a different log in case of a null `testCaseStartEventArgs`?

Reply:
ditto